### PR TITLE
some GIS pkgs

### PR DESCRIPTION
**Pull request by ocefpaf**
Some of those are still using `2to3` and cannot be `noarch` :unamused:

### CODE REVIEW

File: recipes/libpysal/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/l/libpysal/libpysal-{{ version }}.tar.gz
+  sha256: 17b9bcc769a40bd1e5f61d453cd72e60aa440521b6d164c5ac3e8bcda50c3669
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```
Comment (marcelotrevisani) [inline recipes/libpysal/meta.yaml]: hi @ocefpaf , is this the new way in conda to install python projects?

Comment (ocefpaf) [reply recipes/libpysal/meta.yaml]: Yes. @msarahan updated conda-skeleton recently but we haven't update our docs to match that yet.

File: recipes/mgwr/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+test:
+  imports:
+    - mgwr
+
+about:
+  home: https://github.com/pysal/mgwr
+  license: BSD-3-Clause
+  # https://github.com/pysal/mgwr/issues/25
+  # license_file:
```
Comment (marcelotrevisani) [inline recipes/mgwr/meta.yaml]: the liecense file is missing

Comment (ocefpaf) [reply recipes/mgwr/meta.yaml]: Yep. See the GH issue above.

File: recipes/libpysal/meta.yaml (hunk 1 above)
Comment (marcelotrevisani) [reply recipes/libpysal/meta.yaml]: Thanks! :)

**Pull request merged**